apps/judicial-system/backend/src/app/modules/case/case.service.ts

```diff
@@ -414,12 +415,63 @@ export class CaseService {
     }
   }
 
+  private async deliverIndictmentRulingToCourt(
```

Comment:
If I'm not mistaken, the only difference between this function and `deliverIndictmentCourtRecortToCourt` is the selection of which documents to deliver. Consider combining these function and add a parameter for the selection.

Reply:
I would rather not, because that would result in a lot of conditionals within the function: which document to fetch, which error messages to display when something go wrong, which title to use, which file name to use, etc.